applications_tests/lethe-fluid/heat_transfer_vof_phase_change_constrain_solid_domain.output

```diff
@@ -3,90 +3,29 @@ Running on 1 MPI rank(s)...
    Number of degrees of freedom: 4851
    Volume of triangulation:      0.714
    Number of thermal degrees of freedom: 1617
-   Number of VOF degrees of freedom: 1617
 Pressure drop: 0 Pa
 Total pressure drop: 0 Pa
 
-----------------------
```

Comment:
Why is the test case without VOF now? I don't understand, this used to be  VOF test case...